.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@ We will periodically merge updates from the live branch to master to keep master
 
 Documentation Links
 **This link is currently only available for Microsoft Employees** - [Staging review from 'master' branch](https://review.docs.microsoft.com/windows/communitytoolkit/?branch=master)
```

Comment:
We don't have to update this file. But we should modify this anyway and adapt a new process moving forward, so let's not worry about it.

Reply:
For tracking: https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/issues/485